commit cf79716e6636400ae38c37bc8a652b1e522abbba
Author: Quinn Tran <qutran@marvell.com>
Date:   Mon Jun 6 21:46:22 2022 -0700

    scsi: qla2xxx: edif: Synchronize NPIV deletion with authentication application
    
    Notify authentication application of a NPIV deletion event is about to
    occur. This allows app to perform cleanup.
    
    Link: https://lore.kernel.org/r/20220607044627.19563-7-njavali@marvell.com
    Fixes: 9efea843a906 ("scsi: qla2xxx: edif: Add detection of secure device")
    Signed-off-by: Quinn Tran <qutran@marvell.com>
    Signed-off-by: Nilesh Javali <njavali@marvell.com>
    Signed-off-by: Martin K. Petersen <martin.petersen@oracle.com>

diff --git a/drivers/scsi/qla2xxx/qla_edif_bsg.h b/drivers/scsi/qla2xxx/qla_edif_bsg.h
index 110843b13767..0931f4e4e127 100644
--- a/drivers/scsi/qla2xxx/qla_edif_bsg.h
+++ b/drivers/scsi/qla2xxx/qla_edif_bsg.h
@@ -253,4 +253,6 @@ struct aen_complete_cmd {
 
 #define RX_DELAY_DELETE_TIMEOUT 20
 
+#define FCH_EVT_VENDOR_UNIQUE_VPORT_DOWN  1
+
 #endif	/* QLA_EDIF_BSG_H */